packages/react-storage/rollup.config.ts

```diff
@@ -51,6 +51,15 @@ const config = defineConfig([
     },
     plugins: [styles({ mode: ['extract'] })],
   },
+  {
+    input: 'src/storage-browser-styles.ts',
```

Comment:
styles.css is already exported from ui-react-storage, and is an export of the entire UI library's CSS (essentially the same thing if you imported `import '@aws-amplify/ui-react/styles.css';` into your app)

Reply:
Was wondering, since react-ai is new should we follow the same pattern? we currently only h ave the scoped styles of `ai-conversation-styles.css`

Reply:
Got it. So we don't bloat the _src_ directory, think it would make sense to nest these inside a _src/styles_ directory, but that can be done in a follow up